server/statistics/topn_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package statistics
+
+import (
+	"math/rand"
+	"time"
+
+	. "github.com/pingcap/check"
+)
+
+var _ = Suite(&testTopNSuite{})
+
+type testTopNSuite struct{}
+
+type item struct {
+	id    uint64
+	value float64
+}
+
+func (it *item) ID() uint64 {
+	return it.id
+}
+
+func (it *item) Less(than TopNItem) bool {
+	return it.value < than.(*item).value
+}
+
+func (s *testTopNSuite) TestPut(c *C) {
+	const TOTAL = 10000
```

Comment:
TOTAL ->Total
Go generally does not use all-caps constants.

Reply:
Fixed.